common/reviews/api/rush-lib.api.md

```diff
@@ -368,7 +368,7 @@ export class LockStepVersionPolicy extends VersionPolicy {
     // @internal
     get _json(): ILockStepVersionJson;
     get mainProject(): string | undefined;
-    get nextBump(): BumpType;
+    get nextBump(): BumpType | undefined;
```

Comment:
This is technically a breaking change. I'm not sure anyone is using this API, though.

Reply:
fair enough, in runtime it won't break unless you remove the `nextBump` from your policies after the update but it may break some tooling that uses TypeScript.
How do we proceed with it then?

Reply:
I think it's okay as-is.